packages/upset/src/types.ts

```diff
@@ -84,14 +84,22 @@ export type ContextMenuInfo = {
   items: ContextMenuItem[],
 }
 
+/**
+ * Raw data object for an UpSet plot.
+ * This is used to generate the processed data. Column annotations are inferred from the data types.
+ */
+export interface UpsetItem {
+  [key: string]: any;
```

Comment:
Can you replace `any` with `string | number | boolean` or are there other possible types?